js/src/imported_types.ts

```diff
@@ -0,0 +1,1453 @@
+// Auto-generated file -- do not modify
```

Comment:
#nit "imported_types" may be better as "generated_types" and have the `// Auto-generated file` maybe point to the sha/version of the api/version and/or tool that created this